client/change.go

```diff
@@ -78,16 +78,21 @@ type changeAndData struct {
 	Data map[string]*json.RawMessage `json:"data"`
 }
 
-// Change fetches information about a Change given its ID
+// Change fetches information about a Change given its ID.  It can
+// return both change and ErrRebooting if the daemon transmits that it
+// is about to reboot the system.
```

Comment:
It feels a bit unsettling to be doing this here. Seems a bit arbitrary to return the error just on that one operation. The reason why this should fail or not fail is the same for all operations.. depends on the context on the call site rather than on the API itself. That's even more true considering that we are in fact already transforming this back to nil down the chain. So we really just want to take a look.

I have an alternative proposal below.